### DISCUSSION TITLE
Getting ERR_SSL_PROTOCOL_ERROR after deploymeny

### DISCUSSION THREAD

**Discussion by alpuluocak** [vercel/community#5690] [Help] [answered]
### Summary

I have successfully deployed my app on Vercel but now getting ERR_SSL_PROTOCOL_ERROR. 

The backend of my app is a uvicorn server running on EC2 instance. Anyone can shed some light on how to solve this error? Isn't Vercel supposed to generate the SSL certificate via Let's Encrypt?

<img width="838" alt="Screenshot 2024-01-31 at 14 01 08" src="https://github.com/vercel/community/assets/21222270/8136a0d0-5f8e-4e55-8bc4-63714c6ffe73">

<img width="627" alt="Screenshot 2024-01-31 at 14 03 27" src="https://github.com/vercel/community/assets/21222270/52f9110a-b49a-4808-ae50-2f97c8ddb709">

### Example

https://interview-bot-front-end.vercel.app/

### Steps to Reproduce

prod link: https://interview-bot-front-end.vercel.app/

You can try by clicking the 'Clear History' button and inspect

**Comment by swarnava** [comment, marked as answer]
Hi,

We only issue certificate for Vercel hosted domain and application. You need to issue certificate for domain 51.20.32.239 which is hosted in different infrastructure / EC2 instance.
